src/mobile/src/components/Banner/BannerText.jsx

```diff
@@ -25,19 +25,38 @@ const styles = StyleSheet.create({
   },
 });
 
+async function fetchStudentQuotes() {
+  const { data, error } = await supabase.from('quotes').select('*');
+
+  if (error) {
+    throw new Error(error.message);
+  }
+
+  return data;
+}
+
 const BannerText = () => {
-  const [studentQuote, setStudentQuote] = useState(quotes[0]);
+  const [studentQuotes, setStudentQuotes] = useState({});
 
   useEffect(() => {
-    setStudentQuote(quotes[Math.floor(Math.random() * quotes.length)]);
+    fetchStudentQuotes()
+      .then((quotes) => {
+        const randomIndex = Math.floor(Math.random() * quotes.length);
+        setStudentQuotes(quotes[randomIndex]);
+        return quotes;
+      })
+      .catch((err) => console.log(err));
   }, []);
 
   return (
     <View style={styles.container}>
       <Text style={styles.logo}>Telescope</Text>
-      <Pressable style={styles.studentQuote} onPress={() => Linking.openURL(studentQuote.url)}>
-        <Text style={styles.studentQuoteText}>"{studentQuote.quote}"</Text>
-        <Text style={styles.studentQuoteText}> {studentQuote.author}</Text>
+      <Pressable
+        style={styles.studentQuote}
+        onPress={() => Linking.openURL(studentQuotes.blog_url)}
+      >
+        <Text style={styles.studentQuoteText}>"{studentQuotes.quote}"</Text>
+        <Text style={styles.studentQuoteText}> {studentQuotes.author_name}</Text>
```

Comment:
Might need some checking here, when `studentQuotes` is undefined, it shows some 
![image](https://user-images.githubusercontent.com/58233223/160516609-f1e28893-bdb0-49e0-98d1-fc37ce4eb029.png)

Reply:
have you got any data into the table yet ? I should have render it on condition

Reply:
Not yet, but what if there is no data. It will crash

Reply:
You need to add some quotes to the table first

Reply:
sure working on it

Reply:
So ideally this shouldn't happen

Reply:
Just in case.